app/src/main/res/navigation/organiser_mobile_navigation.xml

```diff
@@ -25,4 +29,15 @@ app:startDestination="@+id/organiser_myEvents_page">
     android:label="@string/title_profile"
     tools:layout="@layout/organiser_fragment_profile"
     android:background="@color/dark_purple"/>
+
+    <fragment
+        android:id="@+id/ViewEntrantsFragment"
```

Comment:
I think were using snake case for most our ids, it would make sense to follow this convention.

Reply:
Oop must've missed that. I'll fix it now